src/containers/Dev.jsx

```diff
@@ -0,0 +1,20 @@
+import React from 'react';
```

Comment:
You should rename this file from `Dev.jsx` to `DevOpsProject.jsx`. File names should match the name of the component they contain